test/commute.js

```diff
@@ -17,7 +16,26 @@ describe('commute', function() {
   });
 
   it('works on Algebraic Data Types such as "Maybe"', function() {
```

Comment:
Or should we perhaps move this to ramda-fantasy? Is there a good use case for `commute` with arrays?